wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/DefaultWorkspaceValidator.java

```diff
@@ -32,9 +32,9 @@
  */
 @Singleton
 public class DefaultWorkspaceValidator implements WorkspaceValidator {
-  /* should contain [3, 20] characters, first and last character is letter or digit, available characters {A-Za-z0-9.-_}*/
+  /* should contain [3, 100] characters, first and last character is letter or digit, available characters {A-Za-z0-9.-_}*/
   private static final Pattern WS_NAME =
-      Pattern.compile("[a-zA-Z0-9][-_.a-zA-Z0-9]{1,18}[a-zA-Z0-9]");
+      Pattern.compile("[a-zA-Z0-9][-_.a-zA-Z0-9]{1,98}[a-zA-Z0-9]");
```

Comment:
What about fixes for corresponding tests like [this one](https://github.com/garagatyi/che/blob/wsName100/wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/DefaultWorkspaceValidatorTest.java#L88)?

Reply:
nice catch! Fixed